packages/hardhat-core/test/internal/hardhat-network/console-log/test.ts

```diff
@@ -0,0 +1,73 @@
+import { assert } from "chai";
```

Comment:
Personal preference, feel free to ignore, but I'd rather name this file `hardhat-network/console-log.ts` than use a directory with a single file. Super weak preference though.

Reply:
Done